public/icon.svg

```diff
@@ -0,0 +1,15 @@
+<svg width="512" height="512" viewBox="0 0 512 512" fill="none" xmlns="http://www.w3.org/2000/svg">
+ <style>
+    #letterform-p { fill: #353533; }
+    #letterform-d { fill: #F58321; }
+    #letterform-c { fill: #1177BD; }
+    @media (prefers-color-scheme: dark) {
+      #letterform-p,
+      #letterform-d,
+      #letterform-c { fill: #FFFFFF; }
```

Comment:
Fancy!

Reply:
Right?? The [article](https://web.dev/building-an-adaptive-favicon/) linked from the commit is a good read.